docs/src/configuration.md

```diff
@@ -69,6 +69,37 @@ and the user-provided one are used in the user configuration if
 `stylix.base16Scheme` is not changed in the user config. If that is the case,
 only the user override is used.
 
+### Accessing
+
+When passing colors to unsupported targets, it is possible to access generated
+color scheme through `lib.stylix.colors`. It contains an attrset, populated by
```

Comment:
```suggestion
When passing colors to unsupported targets, it is possible to access values from the configured
color scheme through `lib.stylix.colors`. It contains an attrset, populated by
```

The color scheme may have been be configured manually, rather than generated.